src/com/javabasic/lesson14oop/RealEstateTax.java

```diff
@@ -0,0 +1,21 @@
+package com.javabasic.lesson14oop;
+
+public class RealEstateTax {
+
+    int realEstateArea;
```

Comment:
don't miss modificator, should be private or protected (for child classes). Also, determine if this field is mutable or not, more safe is immutable fields. So it can be also final.

Reply:
Done